docs/source/examples/plots.rst

```diff
@@ -178,7 +170,6 @@ Examples to illustrate the use of :class:`.GraphScene` in manim.
 The following example illustrates how to draw parametric function plots.
```

Comment:
I can change it to : The following example illustrates how to draw parametric function plots, which does not use `GraphScene`

Reply:
I meant to say that it's awkward that this example has this explanatory line, but none of the the others do. I think changing the name could have the same effect. Maybe `ParametriFunctionWithoutGraphScene`?

Reply:
hmm, also possible, but for me having such a long class name is even more awkward than having some text annotations outside the examples.
And we don't know yet, probably there will come more text annotations in the future.

Reply:
+1 for longer but more explicit class names; I don't feel that long class names are awkward -- especially because these scenes are meant to illustrate something; the names are not something that users have to type regularly.

Reply:
> I don't feel that long class names are awkward -- especially because these scenes are meant to illustrate something

In general I agree on that. 
In this specific case I would like to emphasize that this example is a normal Scene, not a GraphScene like the others, and it feels kind of wrong to me to indicate this in a class name.
However, If both of you prefer ParametriFunctionWithoutGraphScene , it's fine for me that we change it

Reply:
Yes, it sounds like we both would rather the longer class name.